piku_dashboard/templates/app_config.html

```diff
@@ -7,27 +7,58 @@
   <h2>{{appid}} Config</h2>
 </section>
 <section>
-<form method="post" action="{{ url_for('app_config', appid=appid) }}">
-<table>
-<thead>
-  <tr>
-    <th>Name</th>
-    <th>Value</th>
-  </tr>
-</thead>
-<tbody>
-{% for k,v in app_config.items() %}
-  <tr>
-      <td><label for="{{ k }}">{{ k }}</label></td>
-      <td><input type="text" name="{{ k }}" id="{{ k }}" value="{{ v }}"></td>
-  </tr>
-{% endfor %}
-</tbody>
-</table>
-<hr>
-<button type="submit">Save Config</button>
+  <form method="post" action="{{ url_for('app_config', appid=appid) }}">
+    <table>
+      <thead>
+      <tr>
+        <th>Name</th>
+        <th>Value</th>
+        <th></th>
+      </tr>
+      </thead>
+      <tbody>
+      {% for k,v in app_config.items() %}
+      <tr>
+        <td><label for="{{ k }}">{{ k }}</label></td>
+        <td><input type="text" name="{{ k }}" id="{{ k }}" value="{{ v }}"></td>
+        <td>
+          <!-- can't nest forms :( -->
+          <!-- {{ url_for('remove_app_config', appid=appid, config=k) }} -->
+          <!-- <button>DELETE</button>-->
```

Comment:
do we need this commented code? 
I'm not sure how useful to a future maintainer this would be.

Reply:
I forgot to delete it! That's done now.